### PR TITLE
Add test for successful Ulimit Parse.

### PR DESCRIPTION
**Pull request by blatyo**
Signed-off-by: Allen Madsen blatyo@gmail.com

### CODE REVIEW

**Comment by duglin** [comment]
LGTM
But I am curious - was this an issue someplace or just being proactive?

**Comment by blatyo** [comment]
Noticed it when fixing #11769

**Comment by jfrazelle** [comment]
LGTM

File: pkg/ulimit/ulimit_test.go [hunk 1]
```diff
@@ -2,6 +2,13 @@ package ulimit
 
 import "testing"
 
+func TestParseValid(t *testing.T) {
+	u1 := &Ulimit{"nofile", 1024, 512}
+	if u2, _ := Parse("nofile=512:1024"); u1 == u2 {
```
Comment (LK4D4) [inline pkg/ulimit/ulimit_test.go]: I'm not sure if it is right. You comparing pointers to `Ulimit` which obviously not equal and that's why your condition `u1 == u2` is true. You should use `*u1 != *u2` I think.

File: pkg/ulimit/ulimit_test.go [hunk 2]
```diff
@@ -2,6 +2,13 @@ package ulimit
...
+		t.Fatalf("expected %s, but got %s", u1.String(), u2.String())
```
Comment (LK4D4) [inline pkg/ulimit/ulimit_test.go]: `String` method automatically called by `%s`, also maybe `%q` better.

**Comment by LK4D4** [comment]
Lol, sorry, I'll create issue about my comments.

**Pull request merged**